### DISCUSSION TITLE
What looks like a duplication in a file

### DISCUSSION THREAD

**Discussion by egpd1701** [flybywiresim/aircraft#6403] [Help]
Hi - got a question about 1 line in the code on github. Probably showing my ignorance here and not really understanding the code enough - I noticed on 
a32nx/src/systems/a320_systems_wasm/src/lib.rs

line 32 & line 33 seem to be the same - it could be ok but it doesn't look right going by the single DC bus entries

As i said, probably nothing wrong but just doesn't look right

**Comment by tracernz** [comment]
cc @davidwalschots

**Comment by davidwalschots** [comment]
Thanks for spotting that error! Luckily it won't break anything, as it will simply replace the configuration of the previous line. However, I agree with you that it doesn't look right! I'll make sure to remove the duplicate line in a future pull request.
